zcutil/build.sh

```diff
@@ -35,5 +35,5 @@ PREFIX="$(pwd)/depends/x86_64-unknown-linux-gnu/"
 
 make "$@" -C ./depends/ V=1 NO_QT=1
 ./autogen.sh
-./configure --prefix="${PREFIX}" --with-gui=no "$LCOV_ARG" CXXFLAGS='-Wno-deprecated-declarations -Werror -Og -g'
+./configure --prefix="${PREFIX}" --with-gui=no "$LCOV_ARG" CXXFLAGS='-Wno-deprecated-declarations -Wno-placement-new -Wno-terminate -Werror -Og -g'
```

Comment:
This concerns me because these kinds of placement new warnings can reveal potential memory corruption bugs. Meanwhile, boost is full of these warnings, so that's quite unfortunate.

I accept this for now. If we could disable this warning for only boost includes, then we could verify our own software much better. Yet, we'd still be open to memory corruption bugs in boost losing user funds, so ideally boost would be patched to fix such warnings.